src/main/java/org/versly/rest/wsdoc/AnnotationProcessor.java

```diff
@@ -829,6 +830,7 @@ String jsonSchemaFromTypeMirror(TypeMirror type) {
         if (dtoClass != null) {
             try {
                 ObjectMapper m = new ObjectMapper();
+                m.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
```

Comment:
This informs the ObjectMapper to use the toString() method to generate JSON schema enum names for DTOs.